src/DeepCopy/Matcher/Doctrine/DoctrineProxyMatcher.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace DeepCopy\Matcher\Doctrine;
+
+use DeepCopy\Matcher\Matcher;
+
+/**
+ * Match a specific property of a specific class
+ */
+class DoctrineProxyMatcher implements Matcher
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function matches($object, $property)
+    {
+        return ($object instanceof \Doctrine\Common\Persistence\Proxy);
```

Comment:
Will attend to that and the docs!

I have no idea how to write a test for this though :-s